site/includes/sign-on-off.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+	<div class="row">
+{{#is signed "off"}}
```

Comment:
Not sure why the link isn't working for you. Here's a screenshot:
![screenshot-clegc-gckey gc ca 2015-07-28 15-27-33](https://cloud.githubusercontent.com/assets/1810242/8941410/53dbf40a-353d-11e5-8702-7f694bb326ac.png)

And in #6232, I only include a state for signing out. That's because when signing in, the user should be presented with a sign-in page, not have to find the sign-in at the top of the page.